backend/core/src/seeder/seeds/jurisdictions.ts

```diff
@@ -43,6 +46,7 @@ export const defaultJurisdictions: JurisdictionCreateDto[] = [
     emailFromAddress: "Detroit Housing",
     rentalAssistanceDefault:
       "Housing Choice Vouchers, Section 8 and other valid rental assistance programs will be considered for this property. In the case of a valid rental subsidy, the required minimum income will be based on the portion of the rent that the tenant pays after use of the subsidy.",
+    enablePartnerSettings: false,
```

Comment:
Since `enablePartnerSettings` is nullable, one of the seeds should probably not set it at all.